scenemanager/src/main/java/io/pixsight/scenemanager/InflateOnDemandLayout.kt

```diff
@@ -0,0 +1,82 @@
+package io.pixsight.scenemanager
```

Comment:
Maybe we should add an additional example in the README for this feature

Reply:
Oui on va revoir le README.